corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
good question. the reason I didn't leave it in was that it seems like celery processes should be able to share the engine in the same way that the web requests should be able to, so wasn't sure if this needed to clean up after itself.

that said, I don't actually know how the threading / execution model of celery works to know whether this should be treated more like a single process or a shared process. @snopoke do you know? otherwise I can read up on it.